rotkehlchen/tests/unit/decoders/test_aave_v3.py

```diff
@@ -960,3 +960,98 @@ def test_aave_v3_events_with_approval(database, polygon_pos_inquirer, polygon_po
         ),
     ]
     assert events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('scroll_accounts', [['0x76111D2841b41B15e6F07fBae4796a82438D9c90']])
+def test_aave_v3_withdraw_eth(database, scroll_inquirer, scroll_accounts) -> None:
+    """Test that withdrawing ETH from Aave gets decoded properly"""
+    tx_hash = deserialize_evm_tx_hash('0x65cd06fd54a10052c3d9084d14d28c06e2bb328b1ec39730fab9284cb529d068')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=scroll_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp, gained_amount, withdrawn_amount, gas_fees = TimestampMS(1716738746000), '0.000000021776581852', '0.010000021776581852', '0.000058164147479909'  # noqa: E501
+    weth_gateway = string_to_evm_address('0xFF75A4B698E3Ec95E608ac0f22A03B8368E05F5D')
+    expected_events = [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.SCROLL,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_fees)),
+            location_label=scroll_accounts[0],
+            notes=f'Burned {gas_fees} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=58,
+            timestamp=timestamp,
+            location=Location.SCROLL,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.APPROVE,
+            asset=EvmToken('eip155:534352/erc20:0xf301805bE1Df81102C957f6d4Ce29d2B8c056B2a'),
+            balance=Balance(amount=FVal('115792089237316195423570985008687907853269984665640564039457.574007891353058083')),
+            location_label=scroll_accounts[0],
+            notes='Set aScrWETH spending approval of 0x76111D2841b41B15e6F07fBae4796a82438D9c90 by 0xFF75A4B698E3Ec95E608ac0f22A03B8368E05F5D to 115792089237316195423570985008687907853269984665640564039457.574007891353058083',  # noqa: E501
+            counterparty=None,
+            address=weth_gateway,
+        ), EvmEvent(
```

Comment:
This is wrong. Multiple things.

1. The Receive (IN) event comes first while it should be after the OUT.